Source/Plugin.BLE/Windows/BleImplementation.cs

```diff
@@ -20,6 +20,26 @@ public class BleImplementation : BleImplementationBase
         public static BluetoothCacheMode CacheModeGetCharacteristics { get; set; } = BluetoothCacheMode.Cached;
         public static BluetoothCacheMode CacheModeGetServices { get; set; } = BluetoothCacheMode.Cached;
 
+        private static int? runtimeBuildNumber = null;
+
+        /// <summary>
+        /// Get the Windows runtime build number
+        /// Eg running on Windows 10 gives 19041 even though it is executing an application compiled as eg net8.0-windows10.0.22621.0
+        /// ref https://learn.microsoft.com/en-us/windows/release-health/release-information
+        /// ref https://learn.microsoft.com/en-us/windows/release-health/windows11-release-information
+        /// </summary>
+        public static int OsRuntimeBuildNumber
+        {
+            get
+            {
+                if (runtimeBuildNumber is null)
+                {
+                    runtimeBuildNumber = Environment.OSVersion.Version.Build;
+                }
+                return runtimeBuildNumber.Value;
+            }
+        }
```

Comment:
To be honest, I don't see much of a point in adding this property.
Logically, it is not related to BLE at all, but is simply a general property of the OS.
Practically, one could just query `Environment.OSVersion.Version.Build` directly, which is simpler (and IMHO preferable, unless there is a good reason for adding a new property in BleImplementation).

Reply:
The reason is that I were inspired from my own project, where I need a solution for .net framework.
beacuse Environment.OSVersion.Version.Build does not work on .Net framework
So I needed this special method below.,
I agree to simplify now we can use Environment.OSVersion.Version.Build
```
  public static int CurrentBuild()
  {
      var reg = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Microsoft\Windows NT\CurrentVersion");
      var currentBuildStr = (string)reg.GetValue("CurrentBuild");
      var currentBuild = int.Parse(currentBuildStr);
      return currentBuild;
  }
```

Reply:
> The reason is that I were inspired from my own project, where I need a solution for .net framework. beacuse Environment.OSVersion.Version.Build does not work on .Net framework So I needed this special method below.

Thanks for the comment. Note: For Android version checks in Plugin.BLE we also use a slightly different mechanism on .NET 6 and above, see e.g. here:

https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/blob/c894e46d7b51c3ea4debb10320483b08eea17a32/Source/Plugin.BLE/Android/Adapter.cs#L211-L215

IIRC, then `Build.VERSION.SdkInt` should also work with the latest .NET versions, but `OperatingSystem.IsAndroidVersionAtLeast` is better at removing deprecation warnings in such conditional code.

> I agree to simplify now we can use Environment.OSVersion.Version.Build

Thanks for updating this. Looks much better to me now :)